### PR TITLE
fix:  Update readme installation instructions

### PR DESCRIPTION
**Pull request by GordonSmith**
Bump versions to get latest security fixes

Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
(Lorraine requested to review via email)

**Review by lorraineachapman — APPROVED** [review]
Readme looks ok to me

File: res/index.min.js [hunk 1]
```diff
@@ -1 +1 @@
...gleProperties=function(){var e=s.Dermatology.prototype.toggleProperties.apply(this,arguments);return this._showProperties||this.wuDashboard.submitPersist(),e},r.prototype.reset=function(){var e=this;this.wuDashboard.resetPersist().then((function(t){a.cache({}),delete e._prevEspUrl,e.render((function(e){}))}))},r.prototype.enter=function(){s.Dermatology.prototype.enter.apply(this,arguments);var e=this;this._resetButton=(new o.Button).id(this.id()+"_reset").value("Reset").on("click",(function(){e.reset()})),this._toolbar.widgets([this._resetButton,this._propsButton])},r.prototype.update=function(){if(s.Dermatology.prototype.update.apply(this,arguments),this._prevEspUrl!==this.espUrl()){this._prevEspUrl=this.espUrl(),this.espCache()&&a.cache(JSON.parse(this.espCache()));var e=this;this.wuDashboard=new m(this.espUrl()),this.wuDashboard.createGrid().then((function(t){e.widget(t).render((function(t){e.wuDashboard.refresh()}))}))}},r}(s.Dermatology);w.prototype._class+=" BundleDermatology",w.prototype.publish("espUrl",null,"string","ESP Url"),w.prototype.publish("espCache",null,"string","ESP Cache"),e.BundleDermatology=w,e.WUDashboard=m,Object.defineProperty(e,"__esModule",{value:!0})}));
```
Comment (richardkchapman) [inline res/index.min.js:1]: This looks rather like generated code to me - if so, should it really be checked in?

**Review by richardkchapman — CHANGES_REQUESTED** [review]
One question

File: res/index.min.js (hunk 1 above)
Comment (GordonSmith) [reply res/index.min.js:1]: Yes - its part of the bundle and this is where `ecl bundle` pulls the sources from...

**Review by richardkchapman — APPROVED** [review]

**Pull request merged**